docs/administration.rst

```diff
@@ -52,6 +52,41 @@ address will not be found in the database.
     $ b2share access show -e <email_address_of_user>
 
 
+Communities
+-----------
+
+B2SHARE has special command-line interface (CLI) commands to create and edit
+communities.
+
+To **list all communities**, run:
+
+.. code-block:: console
+
+    $ b2share communities list
+
+To **create a community**, make sure that B2SHARE_UI_PATH is defined in your
+environment and points to the b2share ``.../webui/app`` folder, and
+then run:
+
+.. code-block:: console
+
+    $ b2share communities create <name> <description> <logo_file_name>
```

Comment:
Looking at the code above it looks like the name of the logo file name is relative to the directory `img/communities`. This needs to be explained too or the user might try with absolute or relative paths.